observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,109 @@ You can use the {TempoName} xref:../../observability/otel/otel-forwarding-teleme
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="distr-tracing_3-5_{context}"]
+== Release notes for {DTProductName} 3.5
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+////
+[id="distr-tracing_3-5_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* ???
+////
+
+[id="distr-tracing_3-5_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} 3.5 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.15.3].
+
+[NOTE]
+====
+The {TempoName} 3.5 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.7.1.
+====
+
+[id="distr-tracing_3-5_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* With this update, you can configure the Tempo backend services to report the internal tracing data by using the OpenTelemetry Protocol (OTLP).
+* With this update, the `traces.span.metrics` namespace becomes the default metrics namespace on which the Jaeger query retrieves the Prometheus metrics. The purpose of this change is to provide compatibility with the OpenTelemetry Collector version 0.109.0 and later where this namespace is the default. Customers who are still using an earlier OpenTelemetry Collector version can configure this namespace by adding the following field and value: `spec.template.queryFrontend.jaegerQuery.monitorTab.redMetricsNamespace: ""`.
+
+[id="distr-tracing_3-5_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fix:
+
+////
+* Before this update, the `oauth-proxy` container did not have any compute resources assigned to it. With this update, the container correctly sets the resources as specified in the `TempoStack` custom resource.
+////
+
+* Before this update, the {TempoOperator} failed when the `TempoStack` custom resource had the `spec.storage.tls.enabled` field set to `true` and used an Amazon S3 object store with the Security Token Service (STS) authentication. With this update, such a `TempoStack` custom resource configuration does not cause the {TempoOperator} to fail.
+
+[id="distr-tracing_3-5_jaeger-release-notes_{context}"]
+=== {JaegerName}
+
+The {JaegerName} 3.5 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/jaeger-operator-bundle/613b50f3a052bb39f2c5f31e/history[{JaegerOperator} Operator 1.62.0].
+
+[NOTE]
+====
+The {JaegerName} 3.5 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.62.0.
```

Comment:
A correction here. The 3.5 release uses Jaeger 1.65.0

Reply:
Thank you. Fixed.
I assume the Operator version also becomes 1.65.0.